src/main/kotlin/org/eclipse/tractusx/bpdmcertificatemanagement/service/CertificateService.kt

```diff
@@ -124,4 +118,26 @@ class CertificateService(
         }
     }
 
+    private fun validateCertificateBeforeProcess(certificateDocumentRequestDto: CertificateDocumentRequestDto) {
+        validateBPNFormat(certificateDocumentRequestDto.businessPartnerNumber, "BPNL")
+        certificateDocumentRequestDto.issuer?.let { validateBPNFormat(it, "BPN") }
```

Comment:
The Issuer name has to start with BPN?

Reply:
Yes, as per the requirement datatype given for issuer is BPN